src/components/main-form.js

```diff
@@ -217,199 +173,275 @@ class MainForm extends React.Component {
+                    <InputGroup>
+                      <Input
+                        name='options.option1.interestRate'
+                        placeholder='APR'
+                        type='number'
+                        value={option1.interestRate}
+                        onChange={this.updateInput}
+                      />
+                      <InputGroupAddon addonType='append'>
+                        <InputGroupText>%</InputGroupText>
+                      </InputGroupAddon>
+                    </InputGroup>
+                  </Col>
+                  <Col>
+                    <label>Mortgage Option 2</label>
+                    <InputGroup>
+                      <Input
+                        name='options.option2.interestRate'
+                        placeholder='APR'
+                        type='number'
+                        value={option2.interestRate}
+                        onChange={this.updateInput}
+                      />
+                      <InputGroupAddon addonType='append'>
+                        <InputGroupText>%</InputGroupText>
+                      </InputGroupAddon>
+                    </InputGroup>
+                  </Col>
+                </Row>
+                <Accordion title='What is the annual percentage rate (APR)?'>
+                  APR measures how much it costs to borrow money.{' '}
+                  <a href='https://www.bankrate.com/glossary/a/apr/'>
+                    Bankrate
+                  </a>{' '}
+                  provides a more in-depth explanation.
+                </Accordion>
+                <Accordion title='Fixed vs Variable Interest Rates'>
+                  This tool assumes your mortgage interest rate is fixed.{' '}
+                  <a href='https://www.valuepenguin.com/loans/fixed-vs-variable-interest-rates'>
+                    Value Penguin
+                  </a>{' '}
+                  has a great article about how fixed and variable rates work.
+                </Accordion>
+              </FormGroup>
+              <FormGroup>
+                <h6>Return on Investment (ROI)</h6>
                 <InputGroup>
                   <Input
-                    name='options.option1.interestRate'
-                    placeholder='APR'
+                    name='investmentRate'
+                    placeholder='ROI'
                     type='number'
-                    value={option1.interestRate}
+                    value={this.state.investmentRate}
                     onChange={this.updateInput}
                   />
                   <InputGroupAddon addonType='append'>
                     <InputGroupText>%</InputGroupText>
                   </InputGroupAddon>
                 </InputGroup>
-              </InputWrapper>
-              <InputWrapper>
-                <label>Mortgage Option 2</label>
+                <Accordion title='How do I invest?'>
+                  {/* TODO: Could this be Vanguard instead? */}
```

Comment:
Please change it to Vanguard in view of the fb poll results

Reply:
😄 oh right, good catch